docs/apb_cli.md

```diff
@@ -156,52 +146,92 @@ be sufficient.
 
 ## Typical Workflows
 
-#### Local Registry
-In order to use the internal OpenShift Docker Registry to source APBs, you must have configured the Ansible Service Broker to use the `local_openshift` type registry adapter. Please see the [config](https://github.com/openshift/ansible-service-broker/blob/master/docs/config.md#local-openshift-registry) section for more information.
+#### Creating and Testing APBs
+In version `2.0.0` and above, `apb` allows you as a developer to develop and test APB images without using an Ansible Service Broker. `apb` allows the user to configure registries to source APBs from and store the list of images locally (in `~/.apb`). Once the images are found, `apb` allows the user to provision the APB directly for testing.
```

Comment:
I would change "store" to "stores".

`apb` allows the user to configure registries to source APBs from and _stores_ the list of images locally (in `~/.apb`).